skeletons/et-utils-package-skeleton/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "TODO: @nzz/et-utils-<package-name>",
```

Comment:
Is it possible to directly set the package name to the string, I give as a parameter with the CLI command?

Reply:
Currently not implemented but the library used for file copies ([replace-in-file](https://github.com/adamreisnz/replace-in-file)) should be able to do exactly that.
I'll look into it, thanks for the nudge!